pkg/session_recording/recordings.jsx

```diff
@@ -450,6 +450,42 @@
     let RecordingList = class extends React.Component {
         constructor(props) {
             super(props);
+            this.sortBy = this.sortBy.bind(this);
+            this.sortList = this.sortList.bind(this);
+            this.drawSortArrow = this.drawSortArrow.bind(this);
+            this.state = {
+                sorting_field: "start",
+                sorting_asc: false,
+            };
+        }
+
+        drawSortArrow() {
```

Comment:
Since the indicator is no longer, strictly speaking, an arrow, perhaps we should rename this to `drawSortDir`, `drawSortIndicator`, or some such.